src/Umbraco.PublishedCache.NuCache/Property.cs

```diff
@@ -66,12 +67,9 @@ internal class Property : PublishedPropertyBase
                 }
                 else
                 {
-                    if (_sourceValues == null)
-                    {
-                        _sourceValues = new Dictionary<CompositeStringStringKey, SourceInterValue>();
-                    }
+                    EnsureSourceValuesInitialized();
```

Comment:
This is now always locking even when it doesn't need to.

Reply:
`EnsureSourceValuesInitialized` does a null check before locking so it's OK 😄